job-service-db/src/main/resources/procedures/changelog.xml

```diff
@@ -199,6 +199,12 @@
                          schemaName="public">
         </createProcedure>
     </changeSet>
+    <changeSet id="create_process_failed_dependent_jobs_proc" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/processFailedDependentJobs.sql"
+                         procedureName="internal_process_failed_dependent_jobs"
+                         schemaName="public">
+        </createProcedure>
+    </changeSet>
```

Comment:
@dgibson-microfocus I know you looked at the rules around this recently.  Remind me whether this is allowed or if it should be in a v3.3 folder.

Reply:
It should be in this changelog, not in it's own v3.3 folder. Having it in here means the latest version of the stored procedure will always be applied on upgrade and avoids any checksum and duplication problems if changed later on.